tools/importer-rest-api-specs/pipeline/task_generate_data_api_definitions_v2.go

```diff
@@ -20,7 +20,7 @@ func (pipelineTask) generateApiDefinitionsV2(serviceName string, apiVersions []m
 	// Generate each of the API Versions for this Service
 	for _, apiVersion := range apiVersions {
 		logger.Debug("Generating Data API Definition in JSON..")
-		yamlGenerator := dataapigeneratorjson.NewForApiVersion(apiVersion.ServiceName, apiVersion.ApiVersion, outputDirectory, swaggerGitSha, resourceProvider, terraformPackageName, logger.Named("Data API Generator"))
+		yamlGenerator := dataapigeneratorjson.NewForApiVersion(apiVersion.ServiceName, apiVersion.ApiVersion, outputDirectory, resourceProvider, terraformPackageName, logger.Named("Data API Generator"))
```

Comment:
```suggestion
		v2Generator := dataapigeneratorjson.NewForApiVersion(apiVersion.ServiceName, apiVersion.ApiVersion, outputDirectory, resourceProvider, terraformPackageName, logger.Named("Data API Generator"))
```

Reply:
Everything within this file is related to the V2 Generator, so on reflection I'm going to remove the version prefix altogether (and make this `apiVersionGenerator` and `serviceGenerator`)